app/Http/Controllers/App/UsersController.php

```diff
@@ -36,58 +48,176 @@ public function add(Request $request)
 
     public function browse(Request $request)
     {
+        return self::json([], 501, ['message' => 'not yet implemented <3']);
         // TODO check privileges
-        $users = User::with('AdserverWallet')->whereNull('deleted_at')->get();
+        $users = User::with('AdserverWallet')->get();
 
         return self::json($users->toArrayCamelize());
     }
 
     public function delete(Request $request, $user_id)
     {
+        return self::json([], 501, ['message' => 'not yet implemented <3']);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the code after this "return". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1763)